_posts/2019-01-09-cockpit-185.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 185
+author: Gundersanne
+date: 2019-01-09
+tags: cockpit linux
+slug: cockpit-185
+category: release
+summary: Responsive dialogs and kubernetes containers included in docker page graphs
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 185.
+
+### Responsive dialogs
+
+Dialogs on the network page and kdump page are now responsive. As are the
+password dialogs on the users page.
+
+{:.side-by-side}
+![network bridge dialog screenshot](/images/network-bridge-dialog-responsive.png)
+![kdump crash dialog screenshot](/images/kdump-crash-dialog-responsive.png)
+![user password dialog screenshot](/images/users-password-dialog-responsive.png)
+
+### Docker: Include kubernetes containers in CPU/memory graphs
+
```

Comment:
This looks a little empty. Perhaps shorten the title to "in graphs", and be a bit more verbose in a sentence? But I don't have an immediate good idea here either.